temporal-sdk/src/main/java/io/temporal/internal/testservice/TestWorkflowMutableStateImpl.java

```diff
@@ -1193,7 +1197,13 @@ private void processFailWorkflowExecution(
     }
 
     if (!Strings.isNullOrEmpty(data.cronSchedule)) {
-      startNewCronRun(ctx, workflowTaskCompletedId, identity, data, data.lastCompletionResult);
+      startNewCronRun(
+          ctx,
+          workflowTaskCompletedId,
+          identity,
+          data,
+          data.lastCompletionResult,
```

Comment:
nit. I know you haven't changed this, but do we really need to pass both data and data.lastCompletionResult separately?

Reply:
We do - at a different site the last completion result comes from another source.